app/controllers/api/assignment_repos_controller.rb

```diff
@@ -9,7 +9,7 @@ class AssignmentReposController < API::ApplicationController
 
     def index
       repos = AssignmentRepo.where(assignment: @assignment).order(:id)
-      paginate json: repos
+      paginate json: repos, roster: @assignment.organization.roster
```

Comment:
Is the `@assignment.organization` always the same as the `@organization` here?

Reply:
Yep, didn't see `@organization`, changed it to use that instead.